### PR TITLE
[Docs] Fixed link to internal-function-calls

### PR DESCRIPTION
**Pull request by xromrom**
Fixed link in function.rst documentation file.

### CODE REVIEW

File: docs/contracts/functions.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ Function parameters can be used as any other local variable and they can also be
   parameter. This functionality is possible if you enable the ABI coder v2
   by adding ``pragma abicoder v2;`` to your source file.
```
Comment (chriseth) [inline docs/contracts/functions.rst]: This is quite outdated - abicoder v2 has been the default for a while. We should convert this note into "until version ... it was not possible to...".

**Review by cameel — COMMENTED** [review]
Thanks for the fix! The change looks good but I agree with @chriseth that it would be good to update the part about ABI encoder too while we're touching this part.

**Comment by xromrom** [comment]
Hi, thanks for the review and comments!
I updated the part about "abicoder v2", @chriseth could you please look once more?

File: docs/contracts/functions.rst [hunk 2]
```diff
@@ -74,13 +74,13 @@ Function parameters can be used as any other local variable and they can also be
...
 .. note::
 
-  An :ref:`external function<external-function-calls>` cannot accept a
-  multi-dimensional array as an input
-  parameter. This functionality is possible if you enable the ABI coder v2
-  by adding ``pragma abicoder v2;`` to your source file.
+  Until version 0.6.0 it was not possible to use a multi-dimensional array or a struct 
+  as an input for an :ref:`external function<external-function-calls>`. 
+  ``abicoder v2`` made it possible and it's been enabled by default since version 0.8.0
+  (before that we had to enable it with ``pragma abicoder v2;``).
```
Comment (Marenz) [inline docs/contracts/functions.rst]: I would not use `we` here but `you` instead, that's more consistent with how the document started addressing the reader, e.g. in line 58 we also use `you`

Other than that, LGTM

**Review by Marenz — CHANGES_REQUESTED** [review]

File: docs/contracts/functions.rst (hunk 2 above)
Comment (Marenz) [reply docs/contracts/functions.rst]: The coding style CI failure is due to trailing spaces in line 77 and 78

**Comment by xromrom** [comment]
CI tests failed because of Z3 library issue:
_-- Found Z3: /usr/lib64/cmake/z3/Z3Config.cmake (found version "4.9.0.0") 
CMake Error at CMakeLists.txt:74 (message):
SMTChecker tests require Z3 4.8.17 for all tests to pass._
  
 Need more time to look into this.

**Comment by xromrom** [comment]
@Marenz thanks for the review. I changed "we" to "you" and removed trailing spaces (CI code style now is ok).
There are 2 failed CI jobs but it seems they were broken not by me.

**Review by Marenz — DISMISSED** [review]

**Comment by xromrom** [comment]
@cameel, @chriseth hi!
Can we merge this PR?

**Comment by Marenz** [comment]
If you do a rebase on the last changes in the `develop` branch the tests will succeed and then we can merge it :)

**Comment by xromrom** [comment]
The code was rebased.

**Review by cameel — DISMISSED** [review]
Looks good, but the review fixes should be squashed into the original commit.

I'll squash and merge.

**Review by cameel — APPROVED** [review]

**Pull request merged**